theory.tex

```diff
@@ -436,15 +436,231 @@ \subsubsection{Modeling with Difference Constraints}
 \subsection{ASP and Linear Constraints}
 \label{sec:linear:constraints}
 
-\begin{itemize}
-\item Literature: \cite{jakaosscscwa17a}
-\item URL: \url{http://github.com/potassco/clingoLP}
-\item Description: \clingoM{LP} 
-  \begin{itemize}
-  \item deals with integer and real variables
-  \item relies on \cplex{} and \lpsolve{}, respectively
-  \end{itemize}
-\end{itemize}
+Similar to \clingoM{DL}, the system \clingoM{LP} integrates the theory of linear constraints into ASP.
+%
+This theory deals with constraints of the form $w_1x_1+\dots+w_nx_n\leq k$, 
+where $k$ is a real-valued constant and $x_i$ and $w_i$ are real-valued variables and coefficients, respectively, for $1\leq i\leq n$. 
+%
+The syntactic representative of a linear constraint is a linear constraint atom of the form: 
+\[
+\code{\&sum\text{ }\{$w_1$*$x_1$:$\boldsymbol{L}_1$;\text{ }\dots;\text{ }$w_n$*$x_n$:$\boldsymbol{L}_n$\}\text{ }<=\text{ }$k$} 
+\]
+%
+Here, $w_i$, $k$ and $x_i$ are terms as described in Section~\ref{subsec:lang:gringo} for $1\leq i\leq n$. 
+%
+In particular, $w_i$ and $k$ are strings of the form \code{"$n.m$"} with $n\in\mathbb{Z}$ and $m\in\mathbb{N}$ representing a real number.
+%
+If $m=0$ then we may drop the quotes and just write $n$.
+%
+Terms $x_i$ are internally interpreted as variable names. 
+%
+$\boldsymbol{L}_i$ are tuples of literals and correspond to conditions as introduced in Section~\ref{subsec:gringo:terms} for $1\leq i\leq n$.
+%
+An element of a linear constraint atom belongs to the corresponding sum if its condition holds.
+%
+We may drop a condition if it is a tautology. 
+%
+Note that \clingoM{LP} supports further relations, viz. \code{>=}, \code{<}, \code{>}, \code{=} and \code{!=}.
+%
+Linear constraint atoms may occur in heads and bodies of rules. 
+%
+A formal introduction of \clingoM{LP} is given in \cite{jakaosscscwa17a}. 
+%
+The following description is conform with the \clingoM{LP} system available at \url{http:github.com/potassco/clingoLP}.
```

Comment:
Should this be?  https://github.com/potassco/clingoLP

Reply:
Both work, but use the correct one: https://github.com/potassco/clingoLP

Reply:
Yes, always prefer https!

Reply:
I just took the liberty to fix it.